<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomFyleSystem</class>
 <widget class="QWidget" name="CustomFyleSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditCurrentDir"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonSelectFile">
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeView" name="treeViewFileSystem"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
